packages/devextreme-react-generator/src/generator.ts

```diff
@@ -273,6 +326,28 @@ export function mapWidget(
   const propTypings = extractPropTypings(raw.options, customTypeHash)
     .filter((propType) => propType !== null) as IPropTyping[];
 
+  const customTypeImports: Record<string, Array<string>> = {};
+  const defaultTypeImports: Record<string, string> = {};
+  const wildcardTypeImports: Record<string, string> = {};
+
+  widgetCustomTypes.forEach((t) => {
+    if (typeGenerationOptions?.importOverridesMetadata?.defaultImports?.[t]) {
+      defaultTypeImports[t] = typeGenerationOptions.importOverridesMetadata.defaultImports[t];
+      return;
+    }
+    const customType = customTypes.find((item) => item.name === t);
+
+    const module = typeGenerationOptions?.importOverridesMetadata?.importOverrides?.[t] || (customType && customType.module && `devextreme/${customType.module}`);
+    if (module) {
+      if (typeGenerationOptions?.importOverridesMetadata?.nameConflictsResolutionNamespaces?.[t]) {
+        wildcardTypeImports[module] = typeGenerationOptions
+          .importOverridesMetadata.nameConflictsResolutionNamespaces[t];
```

Comment:
**Minor**
I would extract a variable and give it a explaining name (`value` is just for demonstration)
```suggestion
      var value = typeGenerationOptions?.importOverridesMetadata?.nameConflictsResolutionNamespaces?.[t]
      if (value) {
        wildcardTypeImports[module] = value
```

Reply:
fixed